components/footer/README.md

```diff
@@ -12,68 +12,55 @@ Footers help people find what they need after scrolling to the bottom of a web p
-Unsure? Contact the Design System team to ask a question. Ask HERE.
-
-Consider using a footer with collapsible functionality (Footer – Extended) for sites with additional navigation requirements.
-
-## Rationale
-This footer is the current version. It meets all government standards, branding, and accessibility requirements. By using this header for your website or app, you’ll help ensure that citizens have a consistent experience when using B.C. government online services.
-
-## Behaviour
-* Ensure all links lead to correct pages
-* No changes to text when browser window size changes
+## Requirements:
+* Must appear on all public-facing B.C. Government online content and services
+* Home, Disclaimer, Privacy, Accessibility, Copyright and Contact Us must be present. Other links can be added as required.
 
 Footer Links:
-*	“Site title” returns to home page of external site
-*	“About” explains external site purpose. Provide background and context of who owns, manages, influences the site – aim at being transparent
+*	“Home” returns to home page of external site
 *	“Disclaimer” use core government [disclaimer statement](https://www2.gov.bc.ca/gov/content/home/disclaimer). Any addendums must be vetted by ministry or central legal advisors before being released)
 *	“Privacy” use core government [privacy statement](https://www2.gov.bc.ca/gov/content/home/privacy). Any addendums must be vetted by ministry or central legal advisors before being released.
 *	“Accessibility” use core government [Accessibility statement](https://www2.gov.bc.ca/gov/content/home/accessibility). Any addendums must be vetted by ministry or central legal advisors before being released.
 *	“Copyright” use core government [copyright statement](https://www2.gov.bc.ca/gov/content/home/copyright). Any addendums must be vetted by ministry or central legal advisors before being released. 
 *	“Contact Us” provides contact details for service area or program.
 
+## Behaviour
+1. Ensure all links lead to correct pages
+2. No changes to text size or weighting when browser window size changes
+
 ## Accessibility
-This footer has been built according to WCAG 2.0 AA standards and has been successfully tested for the following:
+This checkbox has been built according to [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/) standards and all government services should strive to meet this level.  This component successfully includes the following accessibility features:
 
-1. Perceivable
-*	Alternative text is provided for non-text content (WCAG 1.1.1)
-*	Colour is not used as the only visual means of conveying information (WCAG 1.4.1)
-*	Visual presentation of text and images has a contrast ration of at least 4.5:1 (WCAG 1.4.3)
-*	Text can be resized without assistive technology up to 200% without loss of content or functionality (WCAG 1.4.4)
-*	Images of text are only used for pure decoration (WCAG 1.4.9)
+### Screenreaders
+* Proper ARIA and ALT labels are included
```

Comment:
Have we included ARIA labels yet? I agree that we should, but not sure if this is incorporated in our components yet.